### PR TITLE
Change service name

### PR DESCRIPTION
**Pull request by sima-zhu**
Doc.Ms toc work needs service name for reading and parsing readme file. Having special character needs special logic to handle with. Keep things simple for automation work

### CODE REVIEW

File: _data/releases/latest/java-packages.csv [hunk 1]
```diff
@@ -59,7 +59,7 @@
 "azure-storage-blob-cryptography","com.azure","12.15.0","","Storage - Blobs Cryptography","Storage","storage","","","client","true","","10/31/2019","active","","","","",""
 "azure-storage-blob-nio","com.azure","","12.0.0-beta.17","Storage - Blobs NIO","Storage","storage","","","client","true","","","","","","","",""
 "azure-storage-common","com.azure","12.15.0","","Storage - Common","Storage","storage","","","client","true","","10/31/2019","active","","","","",""
-"azure-storage-file-datalake","com.azure","12.8.0","","Storage - Files Data Lake","Data Lake Storage (Gen 2)","storage","","","client","true","","03/12/2020","active","","com.microsoft.azure/azure-data-lake-store-sdk","","",""
+"azure-storage-file-datalake","com.azure","12.8.0","","Storage - Files Data Lake","Data Lake Storage","storage","","","client","true","","03/12/2020","active","","com.microsoft.azure/azure-data-lake-store-sdk","","",""
```
Comment (sima-zhu) [inline _data/releases/latest/java-packages.csv:62]: Is this the right service name to use?

Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:62]: @czubair @scottaddie could you review this or suggest someone to review it from the PM team?

Comment (scottaddie) [reply _data/releases/latest/java-packages.csv:62]: Tagging @kyle-patterson, the PM for this service's SDK.

Comment (sima-zhu) [reply _data/releases/latest/java-packages.csv]: Management is naming gen 1. However, can we have simplified service name so we can have our automation work easy.

Comment (kyle-patterson) [reply _data/releases/latest/java-packages.csv]: Looks right to me!

**Review by kyle-patterson — APPROVED** [review]

**Pull request merged**